packages/affiliates/apps/DappMiningRewards.js

```diff
@@ -9,12 +10,17 @@ const Queries = require("../libs/bigquery");
 // This is the main function which configures all data sources for the calculation.
 async function App(config) {
   const web3 = getWeb3();
+  const { version = "v2" } = config;
+  assert(
+    DappMining[version],
+    "Invalid version in dappmining config, must be one of: " + Object.keys(DappMining).join(", ")
+  );
```

Comment:
this allows us to specify a version of dapp mining to run. This is a backward compatible change, so no configs will need to change, unless specifically running v1.